defaults/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 deploy_user: '{{ ansible_ssh_user }}'
 deploy_group: '{{ ansible_ssh_user }}'
+
+laravel_log_rotate: true
```

Comment:
Should specify it means nginx log, not application log. How about `laravel_rotate_nginx_log`?